README.md

```diff
@@ -77,12 +84,30 @@ Keep in mind that both the frontend and backend have hot reload enabled so you w
 
 Following the starter-code's pattern, complete these tasks:
 
-Currently, our restaurant directory maintains a singular list of restaurants. Suppose we now want the ability to create restaurant groups (e.g. "Uni Plaza", "Cafes", "Open Late", etc.). Each group consists of a subset of the restaurants; a restaurant can be part of multiple groups.
+Currently, our restaurant directory maintains a singular list of restaurants. 
+Suppose we now want the ability to create restaurant groups (e.g. "Uni Plaza", "Cafes", "Open Late", etc.). 
 
-A `RestaurantGroup` has these fields: id, name, description, restaurantIds
+A single group can contain multiple restaurants, and a single restaurant can be part of multiple groups.
 
-1. Create REST endpoints for `RestaurantGroup`, supporting CRUD operations
-    * The API responses should include full restaurant information, not just the ids
-    * Deleting a restaurant should result in its removal from all groups it was part of
+A `RestaurantGroup` has these fields: `id`, `name`, `description`, `restaurantIds`
 
+1. Using the existing code as a template, create REST endpoints for `RestaurantGroup`, supporting C.R.U. (Create, Retrieve, Update) operations. 
+    a. To support _creating_ groups, make an endpoint called `/groups` which handles a `POST` request. It will accept a `name`, `description`, and `restaurantIds` as the request parameters to create a new Restaurant Group. It will return an `id` which corresponds to the newly created Restaurant Group if successful
```

Comment:
> request parameters

I think request body? Might help if we include the expected request and response JSON structures here 🙂

Reply:
Good catch! And yeah good idea I'll add that. Were you thinking of the ids just being an array?

Reply:
Yep! Array of Mongo objectIds, e.g. `[5ff7e50a3e569d00810b67f5, 5ff7e50a3e569d00810b67f6]`